### PR TITLE
HW1_Ihor_Holubtsov

### PR DESCRIPTION
**Pull request by Swipp1233**
god bless

### CODE REVIEW

File: cypress/e2e/tests.cy.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import {sign_in_page} from "../selectors/sign_in_page_selectors";
+
+describe('cypress_loginPage_cases', () => {
+
+  before(() => {
+    cy.visit('/')
+    cy.viewport(1920, 1080)
```
Comment (ievgendco) [inline cypress/e2e/tests.cy.js]: It should be moved to the config file

File: cypress/e2e/tests.cy.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    cy.get(sign_in_page.remember_me_checkbox).click()
+    cy.get(sign_in_page.remember_me_checkbox).should('be.not.checked')
+  })
+
+  it('should show disabled by default sign in btn', () => {
+    cy.get(sign_in_page.login_field).should('have.value', '')
+    cy.get(sign_in_page.sign_in_button).should('be.disabled')
+  })
+
+  it('should have \'Don\'t have an account? Sign Up\' clickable link under \'Sign in\' btn', () => {
```
Comment (ievgendco) [inline cypress/e2e/tests.cy.js]: You should use " (double quotes) to wrap the scenario name in the case when the name contains ' (single quote) e.g.:
"should have 'Don't have an account? Sign Up' clickable link under 'Sign in' btn"

File: cypress/e2e/tests.cy.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    cy.get(sign_in_page.dont_have_acc_hyperlink).click()
+    cy.url().should('be.equal', 'http://localhost:3000/signup')
+    cy.go('back')
+  })
+
+  it('should show Cypress copyright link that leads to \'https://www.cypress.io/\'', () => {
```
Comment (ievgendco) [inline cypress/e2e/tests.cy.js]: The same here

File: cypress/selectors/sign_in_page_selectors.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    password_field: '[id=password]',
+    username_placeholder: '[id=username-label]',
+    password_placeholder: '[id=password-label]',
+    username_error_message: '[id="username-helper-text"]',
+    remember_me_checkbox: '[type="checkbox"]',
+    sign_in_button: '[data-test="signin-submit"]',
+    dont_have_acc_hyperlink: '[data-test="signup"]'
+
+
+
```
Comment (ievgendco) [inline cypress/selectors/sign_in_page_selectors.js:13]: Please remove new lines

File: cypress/e2e/tests.cy.js (hunk 2 above)
Comment (Swipp1233) [reply cypress/e2e/tests.cy.js]: fixed

File: cypress/selectors/sign_in_page_selectors.js (hunk 4 above)
Comment (Swipp1233) [reply cypress/selectors/sign_in_page_selectors.js:13]: fixed

File: cypress/e2e/tests.cy.js (hunk 3 above)
Comment (Swipp1233) [reply cypress/e2e/tests.cy.js]: fixed

File: cypress/e2e/tests.cy.js (hunk 1 above)
Comment (Swipp1233) [reply cypress/e2e/tests.cy.js]: fixed

**Review by anatoly-devico — APPROVED** [review]
Approved

**Pull request merged**